doc/nrf/software_maturity.rst

```diff
@@ -201,7 +201,17 @@ The following table indicates the software maturity levels of the support for ea
 
 .. toggle::
 
-  .. sml-table:: bluetooth
+   +------------------------------------------+----------+----------+----------+----------+----------+----------+----------+---------+---------+
+   |                                          | nRF52810 | nRF52811 | nRF52820 | nRF52832 | nRF52833 | nRF52840 | nRF5340  | nRF9160 | nRF9161 |
+   +==========================================+==========+==========+==========+==========+==========+==========+==========+=========+=========+
+   | Bluetooth LE Peripheral/Central          | Supported| Supported| Supported| Supported| Supported| Supported| Supported| \-      | \-      |
```

Comment:
These names need to be in bold, in all the tables. For example: **Bluetooth LE Peripheral/Central** etc..